en/1/08-functions2.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Private / Public Functions
```

Comment:
The title is Private / Public Functions, but the whole lesson is about returning values and function modifiers. The only place we mention this is in the challenge. Maybe we should rename. And also, since in this step we are teaching users returning values, maybe generate_random_dna should at least return a hardcoded value